lib/data-service.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
 const debug = require('debug')('mongodb-data-service:data-service');
+const Connection = require('mongodb-connection-model');
```

Comment:
I need to add a check if the model passed is not a `Connection` to instantiate a new one.